src/core/src/core_logic/PatchAssessor.py

```diff
@@ -84,6 +84,12 @@ def start_assessment(self):
                 self.telemetry_writer.write_event("Security assessment: " + str(sec_packages), Constants.TelemetryEventLevel.Verbose)
                 self.status_handler.set_package_assessment_status(sec_packages, sec_package_versions, "Security")
 
+                if self.package_manager.get_package_manager_setting(Constants.PKG_MGR_SETTING_IDENTITY) == Constants.APT:
+                    security_esm_update_query_success, security_esm_updates, security_esm_updates_versions = self.package_manager.get_security_esm_updates()
+                    if security_esm_update_query_success:
+                        self.telemetry_writer.write_event("Security-ESM assessment: " + str(security_esm_updates), Constants.TelemetryEventLevel.Verbose)
+                        self.status_handler.set_package_assessment_status(security_esm_updates, security_esm_updates_versions, "Security-ESM")
```

Comment:
Good to change this to use the Constants entry instead of inline constant.

Reply:
Done